packages/evm/src/utils/__test__/helpers.test.ts

```diff
@@ -50,6 +51,25 @@ describe('constructSubstrateRecipient', () => {
   });
 });
 
+describe('serializeGenericCallParameters', () => {
+  it('should seriailze parameters correctly', () => {
+    const result =
+      '0x0000000000000000000000000000000000000000000000000000000000000032000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000064';
```

Comment:
looks like extra space

Reply:
All values are 32 byte padded so not sure if that is extra space 🤔

Reply:
everything is fine, it's just a way it shown in github